optimum/exporters/onnx/convert.py

```diff
@@ -223,7 +286,9 @@ def export_pytorch(
         device = torch.device(device)
         if device.type == "cuda" and torch.cuda.is_available():
             model.to(device)
-            dummy_inputs = tree_map(lambda value: value.to(device), dummy_inputs)
+            dummy_inputs = tree_map(
+                lambda value: value.to(device) if isinstance(value, torch.Tensor) else None, dummy_inputs
```

Comment:
Not sure about this, but just checking:
```suggestion
                lambda value: value.to(device) if isinstance(value, torch.Tensor) else value, dummy_inputs
```